### PR TITLE
Added recency data sync information

### PR DESCRIPTION
**Pull request by mugumedavid**

### CODE REVIEW

File: program-workflows/hiv-recency-testing.md [hunk 1]
```diff
@@ -64,9 +64,98 @@ The reporting tools include the national HMIS tools are:
...
+The supported authentication methods are stated in the API description below.
+
+#### API description
+
+The description of the API that receives the data can be found at this link.
+https://github.com/METS-Programme/hie-documentation
+
+
+
+#### Data dictionary
```
Comment (ssmusoke) [inline program-workflows/hiv-recency-testing.md]: @mugumedavid Remove this and add a link to this https://github.com/METS-Programme/hie-documentation/blob/master/reference/recency-surveillance.v1.json

File: program-workflows/hiv-recency-testing.md [hunk 2]
```diff
@@ -64,9 +64,98 @@ The reporting tools include the national HMIS tools are:
...
 * Recency Data Export - CSV file with recency data fields 
 
+
+
 ## Data Sharing with Central Server
 
-This is an automated process that occurs every hour until a successful submission is done in a day. Due to the small numbers involved, all the HTS records are sent each time, a process which will be improved as the program proceeds
+This is an automated process that occurs every hour until a successful submission is done in a day. Due to the small numbers involved, all the HTS records are sent each time, a process which will be improved as the program proceeds. The EMR gives the option to change the scheduled sync frequency.
+
+#### Sync data flow
```
Comment (ssmusoke) [inline program-workflows/hiv-recency-testing.md]: @mugumedavid This needs to be 3 hashes for Heading 3 and all other headings in this section

File: program-workflows/hiv-recency-testing.md (hunk 1 above)
Comment (mugumedavid) [reply program-workflows/hiv-recency-testing.md]: Perhaps then the API documentation should describe the column attributes in more details. For example, `health_unit_sub_county` actually takes the client's address' sub-county and defaults to facility sub-county if the client address information is not available.

File: program-workflows/hiv-recency-testing.md (hunk 2 above)
Comment (mugumedavid) [reply program-workflows/hiv-recency-testing.md]: Actually, the main heading that takes 3 hashes is the one I found "Data Sharing with Central Server" All headings I introduced are under that heading.

Comment (mugumedavid) [reply program-workflows/hiv-recency-testing.md]: Oh okay. I skipped from 2 to 4 hashes. Got it.

File: program-workflows/hiv-recency-testing.md [hunk 3]
```diff
@@ -56,17 +56,54 @@ This data is entered following the steps in [HIV Testing Services Client Card](.
...
+The username is preferred as the national DHIS2 ID. The password is a random set of characters and numbers.
+
+### Sync authentication
+
+The supported authentication methods are stated in the API description below.
+
+### API description
+
+The description of the API that receives the data can be found at this link.
+https://github.com/METS-Programme/hie-documentation
```
Comment (ssmusoke) [inline program-workflows/hiv-recency-testing.md]: @mugumedavid Add the link to the JSON file for Recency as we shall be adding more files to the repository

File: program-workflows/hiv-recency-testing.md [hunk 4]
```diff
@@ -56,17 +56,54 @@ This data is entered following the steps in [HIV Testing Services Client Card](.
 ## Reporting
 
 The reporting tools include the national HMIS tools are:  
-1. HMIS ACP 019 HIV Testing Services Register  
+1. HMIS ACP 018 HIV Testing Services Register
```
Comment (mugumedavid) [inline program-workflows/hiv-recency-testing.md:59]: @ssmusoke I believe this is 018 now (with the new tools??)

File: program-workflows/hiv-recency-testing.md (hunk 1 above)
Comment (ssmusoke) [reply program-workflows/hiv-recency-testing.md]: @mugumedavid that is done during transformation not data collection - I also think that is an incorrect assumption leading to misleading results for hotspot placement

File: program-workflows/hiv-recency-testing.md (hunk 4 above)
Comment (ssmusoke) [reply program-workflows/hiv-recency-testing.md:59]: @mugumedavid  That is correct

File: program-workflows/hiv-recency-testing.md [hunk 5]
```diff
@@ -56,17 +56,54 @@ This data is entered following the steps in [HIV Testing Services Client Card](.
...
+
+
 ## Data Sharing with Central Server
 
-This is an automated process that occurs every hour until a successful submission is done in a day. Due to the small numbers involved, all the HTS records are sent each time, a process which will be improved as the program proceeds
+This is an automated process that occurs every hour until a successful submission is done in a day. Due to the small numbers involved, all the HTS records are sent each time, a process which will be improved as the program proceeds. The EMR gives the option to change the scheduled sync frequency.
+
+### Sync data flow
+
+![](/assets/sync-data-flow.png)
```
Comment (ssmusoke) [inline program-workflows/hiv-recency-testing.md]: @mugumedavid add ALT Text (in the square brackets) and use a relative path to the image this one is broken

**Pull request merged**